packages/eas-json/schema/eas.schema.json

```diff
@@ -334,8 +335,9 @@
                 "macos-big-sur-11.4-xcode-12.5"
               ],
               "markdownEnumDescriptions": [
-                "`macos-monterey-12.6-xcode-14.1`",
-                "`macos-monterey-12.6-xcode-14.2`"
+                "`macos-monterey-12.6-xcode-14.2` for `m-medium` resource class, `macos-monterey-12.6-xcode-14.1` for `intel-medium` resource class",
```

Comment:
Same question as on the docs PR, does this mean we need to update and bump/release this again when we have other silicon resource classes shortly?